index.html

```diff
@@ -198,5 +198,23 @@ <h2 class="aboutLink">LET'S CONNECT</h2>
             </ul>
         </section>
     </section>
+
+    <section class="contactMe">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/mbjeoppw" method="post">
+            <ul>
+                <li>
+                    <input name="user_name" id="name" placeholder="Yeremias" type="text" maxlength="30" required>
+                </li>
+                <li><input name="email" id="email" placeholder="natajayanj14@gmail.com" type="email" required>
+                </li>
+                <li>
+                    <textarea name="comment" id="comment" placeholder="Write your message here" maxlength="500" required></textarea>
+                </li>
+                <button type="submit">Get in touch</button>
+            </ul>
```

Comment:
- [ ] Nice work implementing the submit button successfully, however, you still need to set the proper interactions for this button as shown in the Figma design 🔽 remember that according to the project's [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), you need to stick to the Figma design as much as possible.

![2023-02-22_220755](https://user-images.githubusercontent.com/104319462/220759715-5940e69b-7e94-475a-a127-8b4707290542.png)

Reply:
Thanks for your kind feedback!